doc/command_ref.rst

```diff
@@ -167,7 +167,7 @@ Options
 ``--disableexcludes=[all|main|<repoid>], --disableexcludepkgs=[all|main|<repoid>]``
     Disable the configuration file excludes. Takes one of the following three options:
 
-    * ``all``, disables all configuration file excludes
+    * ``all``, disables all configuration excludes and includes
```

Comment:
Would it make sense to specify that `--disableexcludes` operates on both `excludepkgs` and `includepkgs` before this bullet list? I believe it disables both even for `main` and `repoid` values.